sdk/src/types/block/error.rs

```diff
@@ -79,7 +80,7 @@ pub enum Error {
     InvalidInputKind(u8),
     InvalidInputCount(<InputCount as TryFrom<usize>>::Error),
     InvalidInputOutputIndex(<OutputIndex as TryFrom<u16>>::Error),
-    InvalidBech32Hrp(String),
+    InvalidBech32Hrp(Bech32HrpError),
```

Comment:
Impl from?

Reply:
We could do that oc. On the other hand, there aren't many places that would profit from it tbh.

Reply:
but since it's such a small change, I'll add it anyway.

Reply:
ah damn, this is not a `thiserror::Error`, so not that small, but okay